### PR TITLE
Add security tip for setCookie

### PR DESCRIPTION
**Pull request by FedericoCeratto**

### CODE REVIEW

File: lib/pure/cookies.nim [hunk 1]
```diff
@@ -50,6 +50,7 @@ proc setCookie*(key, value: string, domain = "", path = "",
                 maxAge = none(int), sameSite = SameSite.Default): string =
   ## Creates a command in the format of
   ## `Set-Cookie: key=value; Domain=...; ...`
+  ## .. tip: Cookies can be vulnerable. Consider setting secure=true, httpOnly=true and sameSite=Strict
```
Comment (dom96) [inline lib/pure/cookies.nim]:
```suggestion
  ## `Set-Cookie: key=value; Domain=...; ...`
  ##
  ## .. tip: Cookies can be vulnerable. Consider setting secure=true, httpOnly=true and sameSite=Strict
```

**Review by dom96 — COMMENTED** [review]
I wonder why browsers don't just enable these by default. Guess they break some critical things.

File: lib/pure/cookies.nim (hunk 1 above)
Comment (konsumlamm) [inline lib/pure/cookies.nim]:
```suggestion

  ## .. tip: Cookies can be vulnerable. Consider setting `secure=true`, `httpOnly=true` and `sameSite=Strict`.
```

File: lib/pure/cookies.nim [hunk 2]
```diff
@@ -50,6 +50,9 @@ proc setCookie*(key, value: string, domain = "", path = "",
                 maxAge = none(int), sameSite = SameSite.Default): string =
   ## Creates a command in the format of
   ## `Set-Cookie: key=value; Domain=...; ...`
+  ##
+
+  ## .. tip: Cookies can be vulnerable. Consider setting `secure=true`, `httpOnly=true` and `sameSite=Strict`.
```
Comment (ringabout) [inline lib/pure/cookies.nim:55]: This is not correct and failed to generate docs.
`Warning: RST style: double colon :: may be missing at end of 'tip' [warnRstStyle]`

Comment (ringabout) [reply lib/pure/cookies.nim:55]: So it is better to use warnAsErrors when building our docs for CI.

**Pull request merged**